apollo-runtime/src/main/java/com/apollographql/apollo/ApolloMutationCall.java

```diff
@@ -0,0 +1,41 @@
+package com.apollographql.apollo;
+
+import com.apollographql.apollo.api.Mutation;
+import com.apollographql.apollo.api.OperationName;
+import com.apollographql.apollo.cache.CacheHeaders;
+
+import javax.annotation.Nonnull;
+
+/**
+ * A call prepared to execute GraphQL mutation operation.
+ */
+public interface ApolloMutationCall<T> extends ApolloCall<T> {
+
+  /**
+   * <p>Sets a list of GraphQL query names to be re-fetched once this mutation completed.</p>
```

Comment:
Well, we might want to refetch a query you aren't watching. Like if you have a post feed page, and a child page that creates a post, we might want to refetch the post feed query to update the normalized cache, even if we aren't watching